frontend/src/Pages/AllUser/HomePage/Home.jsx

```
@@ -0,0 +1,11 @@
+import Hero  from "./HeroSection/Hero";
+
+const Home = () => {
+    return (
+        <div>
+            <Hero></Hero>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)